### PR TITLE
Translations for various pages

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-4038
https://tools.hmcts.net/jira/browse/RET-4033
https://tools.hmcts.net/jira/browse/RET-4057
https://tools.hmcts.net/jira/browse/RET-4047

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/locales/cy/translation/your-applications.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+  "Contact about something else": "Cysylltu ynghylch rhywbeth arall",
+  "Order respondent to do something": "Gorchymyn i’r atebydd wneud rhywbeth",
+  "Order a witness to attend": "Gorchymyn i dyst ddod i wrandawiad",
+  "Postpone a hearing": "Gohirio gwrandawiad",
+  "Reconsider judgement": "Ailystyried dyfarniad",
+  "Restrict publicity": "Cyfyngu ar gyhoeddusrwydd",
+  "Strike out all/part of response": "Dileu’r ymateb cyfan neu ran o’r ymateb",
+  "Vary/revoke an order": "Amrywio neu ddiddymu gorchymyn",
+  "Withdraw all/part of claim": "Tynnu fy hawliad yn ôl",
+  "withdraw": "IS THIS REQUIRED Welsh translation required"
```
Comment (lvo700) [inline src/main/resources/locales/cy/translation/your-applications.json]: Is this one still missing?

Comment (brendanmchugh) [reply src/main/resources/locales/cy/translation/your-applications.json]: will remove that

**Review by lvo700 — DISMISSED** [review]

**Review by jip1984 — DISMISSED** [review]

File: src/main/controllers/ApplicationDetailsController.ts [hunk 2]
```diff
@@ -94,7 +95,7 @@ export default class ApplicationDetailsController {
       ...content,
       header,
       selectedApplication,
-      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink),
+      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink, req.url),
```
Comment (marcjudge) [inline src/main/controllers/ApplicationDetailsController.ts:98]: Add welshEnabled feature flag

File: src/main/controllers/RespondToTribunalResponseController.ts [hunk 3]
```diff
@@ -128,7 +129,7 @@ export default class RespondToTribunalResponseController {
     assignFormData(req.session.userCase, this.form.getFormFields());
     res.render(TranslationKeys.RESPOND_TO_TRIBUNAL_RESPONSE, {
       ...content,
-      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink),
+      appContent: getTseApplicationDetails(selectedApplication, translations, downloadLink, req.url),
```
Comment (marcjudge) [inline src/main/controllers/RespondToTribunalResponseController.ts:132]: Add welshEnabled feature flag

File: src/main/controllers/RespondentApplicationDetailsController.ts [hunk 4]
```diff
@@ -91,7 +91,7 @@ export default class RespondentApplicationDetailsController {
       header,
       selectedApplication,
       redirectUrl,
-      appContent: getTseApplicationDetails(selectedApplication, translations, supportingMaterialDownloadLink),
+      appContent: getTseApplicationDetails(selectedApplication, translations, supportingMaterialDownloadLink, req.url),
```
Comment (marcjudge) [inline src/main/controllers/RespondentApplicationDetailsController.ts:94]: Add welshEnabled feature flag

**Review by marcjudge — CHANGES_REQUESTED** [review]

File: src/main/controllers/ApplicationDetailsController.ts (hunk 2 above)
Comment (brendanmchugh) [reply src/main/controllers/ApplicationDetailsController.ts:98]: Not sure if the welshEnabled flag should be added for this controller.  I made a change to the shared getTseApplicationDetails helper function, which required the url to be passed but the applicationDetailsController / respondToTribunalResponse weren't pages being that were translated in this pull request.

**Review by marcjudge — APPROVED** [review]

**Review by jip1984 — APPROVED** [review]

**Pull request merged**